src/classes/CRLP_DefaultConfigBuilder_SVC.cls

```diff
@@ -267,20 +268,56 @@ public class CRLP_DefaultConfigBuilder_SVC {
 
-        CRLP_RollupCMT.FilterRule ruleCloseWonStage4 = new CRLP_RollupCMT.FilterRule(groupClosedWonDonationsNoRTExclusions.recordName, 'Opportunities: Won Only');
+        CRLP_RollupCMT.FilterRule ruleCloseWonStage4 = new CRLP_RollupCMT.FilterRule(groupClosedWonDonationsNoRTExclusions.recordName, 'Opportunities: Won Only (RD)');
         ruleCloseWonStage4.recordName = CRLP_DefaultConfigBuilder.RecordNamePrefix + ruleCloseWonStage4.recordName;
         ruleCloseWonStage4.objectName = 'Opportunity';
         ruleCloseWonStage4.fieldName = 'IsWon';
         ruleCloseWonStage4.operationName = CMT_FilterRule.FilterOperation.EQUALS.name();
         ruleCloseWonStage4.value = 'true';
         groupClosedWonDonationsNoRTExclusions.rules.add(ruleCloseWonStage4);
 
+        // ==============================================================================================================
+        // Default "Payments: Paid" filter group
+        // ==============================================================================================================
+        CRLP_RollupCMT.FilterGroup groupPaymentsPaid = new CRLP_RollupCMT.FilterGroup('Payments: Paid');
+        groupPaymentsPaid.recordName = CRLP_DefaultConfigBuilder.FilterGroup_PaymentsPaid;
+        groupPaymentsPaid.description = 'All paid Payments.';
+
+        CRLP_RollupCMT.FilterRule rulePaid = new CRLP_RollupCMT.FilterRule(groupPaymentsPaid.recordName, 'Payments: Paid Only');
+        rulePaid.recordName = CRLP_DefaultConfigBuilder.RecordNamePrefix + rulePaid.recordName;
+        rulePaid.objectName = 'npe01__OppPayment__c';
+        rulePaid.fieldName = 'npe01__Paid__c';
+        rulePaid.operationName = CMT_FilterRule.FilterOperation.EQUALS.name();
+        rulePaid.value = 'true';
+        groupPaymentsPaid.rules.add(rulePaid);
+
+        // ==============================================================================================================
+        // Default "Payments: Paid" filter group
+        // ==============================================================================================================
+        CRLP_RollupCMT.FilterGroup groupPaymentsWrittenOff = new CRLP_RollupCMT.FilterGroup('Payments: Written Off');
+        groupPaymentsWrittenOff.recordName = CRLP_DefaultConfigBuilder.FilterGroup_PaymentsWrittenOff;
+        groupPaymentsWrittenOff.description = 'All written off Payments.';
+
+        CRLP_RollupCMT.FilterRule ruleWrittenOff = new CRLP_RollupCMT.FilterRule(groupPaymentsWrittenOff.recordName, 'Payments: Written Off Only');
+        ruleWrittenOff.recordName = CRLP_DefaultConfigBuilder.RecordNamePrefix + ruleWrittenOff.recordName;
+        ruleWrittenOff.objectName = 'npe01__OppPayment__c';
+        ruleWrittenOff.fieldName = 'npe01__Written_Off__c';
+        ruleWrittenOff.operationName = CMT_FilterRule.FilterOperation.EQUALS.name();
+        ruleWrittenOff.value = 'true';
+        groupPaymentsPaid.rules.add(ruleWrittenOff);
```

Comment:
Part of the issue with the rule ending up in the wrong group is this line, which adds this new rule to the PaymentsPaid group.